testing/src/test/java/org/motechproject/nms/testing/it/imi/ImiController_OBD_BundleIT.java

```diff
@@ -180,4 +180,28 @@ public void verifyFT200() throws IOException, InterruptedException {
         assertTrue(SimpleHttpClient.execHttpRequest(httpPost, HttpStatus.SC_BAD_REQUEST, expectedJsonResponse,
                 ImiTestHelper.ADMIN_USERNAME, ImiTestHelper.ADMIN_PASSWORD));
     }
+
+    /*
+    * NMS_FT_193 :To check "NotifyFileProcessedStatus" API is rejected in case the file could not
+    * be copied or the file is not available.
+    * NMS_FT_194 : To check "NotifyFileProcessedStatus" API is rejected in case there is an error in checksum
+    * NMS_FT_195 : To check "NotifyFileProcessedStatus" API is rejected in case there is an error in records check.
+    */
+    @Test
+    public void verifyFT193_194_195() throws IOException, InterruptedException {
```

Comment:
yes, this test case is testing only 193, but I have mapped test cases 194 and 195 to this test case only as all the test cases are testing same piece of code except the enum value is different.